packages/react/.storybook/manager.js

```diff
@@ -36,6 +36,7 @@ if (!process.env.DDS_FLAGS_ALL) {
     process.env.DDS_CONTENT_BLOCK_HEADLINES,
     'components-content-block-headlines'
   );
+  css += getCss(process.env.DDS_PRICING_TABLE, 'components-pricing-table');
```

Comment:
I think this is unnecessary because there's not a react version of the pricing table component.